automl/cloud-client/get_model_evaluation.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def get_model_evaluation(project_id, model_id, model_evaluation_id):
+    """Get model evaluation."""
+    # [START automl_get_model_evaluation]
+    from google.cloud import automl
+
+    # TODO(developer): Uncomment and saet the following variables
+    # project_id = 'YOUR_PROJECT_ID'
+    # model_id = 'YOUR_MODEL_ID'
+    # model_evaluation_id = 'YOUR_MODEL_EVALUATION_ID'
+
+    client = automl.AutoMlClient()
+    # Get the full path of the model evaluation.
+    model_evaluation_full_id = client.model_evaluation_path(
+        project_id, 'us-central1', model_id, model_evaluation_id
+    )
+
+    # Get complete detail of the model evaluation.
+    response = client.get_model_evaluation(model_evaluation_full_id)
+
+    print(u'Model evaluation name: {}'.format(response.name))
+    print(u'Model annotation spec id: {}'.format(response.annotation_spec_id))
+    print('Create Time:')
+    print(u'\tseconds: {}'.format(response.create_time.seconds))
+    print(u'\tnanos: {}'.format(response.create_time.nanos / 1e9))
+    print(u'Evaluation example count: {}'.format(
+        response.evaluated_example_count))
+    print(u'Model evaluation metrics: {}'.format(
+        response.translation_evaluation_metrics))
```

Comment:
this should be different for each flavour. please add a couple start/end region tags.

Reply:
alternatively, you can use response.metrics

Reply:
Oops, done. 
metrics won't work, it is a `oneof` field.